### PR TITLE
[AIRFLOW-941] Use defined parameters for psycopg2

### PR DESCRIPTION
**Pull request by bolkedebruin**
This works around https://github.com/psycopg/psycopg2/issues/517 .

Please accept this PR that addresses the following issues:
- https://issues.apache.org/jira/browse/AIRFLOW-941

Testing Done:
- Unit tests pass for Postgres

See: https://travis-ci.org/bolkedebruin/airflow/builds/207717186

### CODE REVIEW

**Comment by vijaykramesh** [comment]
the code looks good to me.  what's the deal with the 2.7 failures? looking in travis I can confirm that it's not that config `read_file` issue that was fixed with [this revert](https://github.com/apache/incubator-airflow/commit/7d95a0dca06505a224e43aa8f767ce8b7d15e99e).

but the travis output is so long I can't really figure out what failed.

File: airflow/hooks/postgres_hook.py [hunk 1]
```diff
@@ -36,10 +36,15 @@ def get_conn(self):
         conn = self.get_connection(self.postgres_conn_id)
         conn_args = dict(
             host=conn.host,
-            user=conn.login,
-            password=conn.password,
-            dbname=self.schema or conn.schema,
-            port=conn.port)
+            dbname=self.schema or conn.schema)
+        # work around for https://github.com/psycopg/psycopg2/issues/517
```
Comment (aoen) [inline airflow/hooks/postgres_hook.py:40]: Can we add a todo to remove this once the issue is fixed?

**Comment by aoen** [comment]
LGTM barring nit above and once Travis passes (or if failures are unrelated)

**Comment by bolkedebruin** [comment]
Py3 passes (it should), py2 failures are ldap3 related as mentioned on the mailing list. Will add todo.

**Comment by bolkedebruin** [comment]
Done. merging.

**Pull request closed without merging**